README.md

```diff
@@ -70,18 +58,29 @@ Options:
   -c, [--config=<config file path>]  # Path to the Tapioca configuration file
                                      # Default: sorbet/tapioca/config.yml
   -V, [--verbose], [--no-verbose]    # Verbose output for debugging purposes
-```
+~~~
 
-## Usage
+## Getting started
 
-### Initialize folder structure
+Execute this command to get started:
 
-Command: `tapioca init`
+~~~sh
+$ bundle exec tapioca init
+      create  sorbet/config
+      create  sorbet/tapioca/config.yml
+      create  sorbet/tapioca/require.rb
+      create  bin/tapioca
+~~~
 
-This will create the `sorbet/config` and `sorbet/tapioca/require.rb` files for you, if they don't exist. If any of the files already exist, they will not be changed.
+This will:
+
+* create the default [configuration file for Sorbet](https://sorbet.org/docs/cli#config-file) and [Tapioca](#Configuration)
+* install the [binstub](https://bundler.io/man/bundle-binstubs.1.html#DESCRIPTION) for Tapioca in your app's `bin/` folder, so that you can use `bin/tapioca` to run commands in your app
```

Comment:
It also creates the require.rb file. Should we mention that and link to somewhere explaining what it is?

Reply:
This is explained in details inside the `#### Manually requiring parts of a gem`. Do you think we should bring it earlier? I think this is only relevant when talking about gems RBI generation.